### ISSUE TITLE
Version 1.5.231 release breaks Microsoft AppCenter.Start() calls which depends on sqlite-net-pcl nuget

### ISSUE THREAD

**Issue by jimmccurdy** [praeclarum/sqlite-net#751]
Version 1.5.231 release breaks Microsoft AppCenter.Start() calls which depends on sqlite-net-pcl nuget.

**Comment by praeclarum** [comment]
Can you give me a repro on how to see the error? Sorry I'm not familiar with what AppCenter is doing...

**Comment by jimmccurdy** [comment]
Just add the AppCenter nugets to a project, and in your App.Xaml.cs, OnStart() method, after base.OnStart():
  
  AppCenter.Start("android=YourAndroidSecret;" +
 "uwp=YourUWPSecret;" +
 "ios=YouriOSSecret;",
 typeof(Analytics), typeof(Crashes), typeof(Push));
  
 and you will receive an sqlite exception.

  
  
  

----------------------------------------
 Return-Path: <bounces+848413-ffb7-jim=facetofacesoftware.com@sgmail.github.com>
Received: from o3.sgmail.github.com (o3.sgmail.github.com [192.254.112.98]) by sm11.internetmailserver.net with SMTP
(version=TLS\Tls12
cipher=Aes256 bits=256);
Wed, 12 Sep 2018 11:38:49 -0700
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com;
h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe;
s=s20150108; bh=cDDv72nySfg23S+stLAW8kBxKDs=; b=fWCPstP/J9MaCwci
yWCJP/C0FsVmejbpLFlWSy1L+ShRl6/qaiVGZ4hcQ8Et4RZ0cXLuehiP1kETqoV7
xd2/gKpl1d1qxGtPu+n4yS5vetTQGNOJCawgF+BMVmwr0Q7Chx6d2kDND81rluGL
ZQk6aMj2YVc9s0eRoYOGubckBbo=
Received: by filter1554p1mdw1.sendgrid.net with SMTP id filter1554p1mdw1-30599-5B995D34-D
2018-09-12 18:38:44.766469558 +0000 UTC m=+508039.482819826
Received: from github-lowworker-05ceafd.cp1-iad.github.net (unknown [192.30.252.40])
by ismtpd0011p1iad2.sendgrid.net (SG) with ESMTP id I-DgAMglTzKyoynVb8Ib4g
for <jim@facetofacesoftware.com>; Wed, 12 Sep 2018 18:38:44.801 +0000 (UTC)
Received: from github.com (localhost [127.0.0.1])
by github-lowworker-05ceafd.cp1-iad.github.net (Postfix) with ESMTP id AFE8A46147F
for <jim@facetofacesoftware.com>; Wed, 12 Sep 2018 11:38:44 -0700 (PDT)
Date: Wed, 12 Sep 2018 18:38:44 +0000 (UTC)

**Comment by leowsouza** [comment]
Hello @jimmccurdy !
I use AppCenter and I wanto to upgrade SQLLite.
Did you found a solution ?

**Comment by jimmccurdy** [comment]
No, but praeclarum did reach out to me, so I am hopeful that a fix is forthcoming. 




On Fri, Sep 14, 2018 at 1:13 PM -0400, "leowsouza" <notifications@github.com> wrote:












Hello @jimmccurdy !

I use AppCenter and I wanto to upgrade SQLLite.

Did you found a solution ?

**Comment by lewcianci** [comment]
I also have this issue - can I do anything to help?

**Comment by praeclarum** [comment]
This should be fixed with 1.6-beta. Please try it out.
